src/pages/CreateAccount/CreateAccount.jsx

```diff
@@ -147,79 +152,76 @@ const CreateAccount = () => {
   };
 
   // TODO: backend connection once auth is finalized
-  const onSubmit = values => {
-    try {
-      const languages = buildLanguagesArray(values);
-
-      const payload = {
-        firstName: values.firstName,
-        lastName: values.lastName,
-        role: values.role,
-        organization: values.organization,
-        birthdate: values.birthdate,
-        email: values.email,
-        phone: values.phone,
-        preferredContactMethod: values.preferredContactMethod,
-        addressStreet: values.addressStreet,
-        addressZip: values.addressZip,
-        addressCity: values.addressCity,
-        addressState: values.addressState,
-        weightLiftingAbility: values.weightLiftingAbility,
-        criminalHistory: values.criminalHistory,
-        criminalHistoryDetails: values.criminalHistoryDetails,
-        duiHistory: values.duiHistory,
-        duiHistoryDetails: values.duiHistoryDetails,
-        completedChowmatchTraining: values.completedChowmatchTraining,
-        foodRunsInterest: values.foodRunsInterest,
-        distributionInterest: values.distributionInterest,
-        canDrive: values.canDrive,
-        willingToDrive: values.willingToDrive,
-        vehicleType: values.vehicleType,
-        distance: values.distance,
-        firstAidTraining: values.firstAidTraining,
-        serveSafeKnowledge: values.serveSafeKnowledge,
-        transportationExperience: values.transportationExperience,
-        movingWarehouseExperience: values.movingWarehouseExperience,
-        foodServiceIndustryKnowledge: values.foodServiceIndustryKnowledge,
-        languages,
-        additionalInformation: values.additionalInformation,
-        availabilities: availability,
-      };
-      console.log(payload);
-      // await AFCBackend.post('/users/', payload);
-    } catch (e) {
-      console.log(e.message);
-    }
-  };
+  const onSubmit = async values => {
+    const languages = buildLanguagesArray(values);
 
-  const onError = (errors, e) => {
-    console.log(errors, e);
+    const payload = {
```

Comment:
first: holy moly that is a lot of items

second: `const payload = {...values, languages, availabilities: availability};`

The `...values` will take all your keys/values from `values` and basically copy them over to `payload` so you don't have to type all those lines out in the future

Reply:
fixed!